pkg/autoscaler/autoscaler/tidb_autoscaler.go

```diff
@@ -24,38 +24,59 @@ import (
 
 func (am *autoScalerManager) syncTiDB(tc *v1alpha1.TidbCluster, tac *v1alpha1.TidbClusterAutoScaler, client promClient.Client) error {
 	if tac.Spec.TiDB == nil {
+		emptyConsecutiveCount(tc, v1alpha1.TiDBMemberType)
```

Comment:
This is just in case? If no config for TiDB, the count will not be populated, right?

Reply:
Or should remote the anno in this case?

Reply:
The spec of AutoScaler config for Tidb could be emptied from an existing status.